{{cookiecutter.project_shortname}}/docker-services.yml

```diff
@@ -0,0 +1,107 @@
+version: '2.3'
+services:
+  app:
+    build:
+      context: ./
+      args:
+        - ENVIRONMENT=DEV
+    image: {{cookiecutter.project_shortname}}
+    environment:
+      - "INVENIO_ACCOUNTS_SESSION_REDIS_URL=redis://cache:6379/1"
+      - "INVENIO_BROKER_URL=amqp://guest:guest@mq:5672/"
+      - "INVENIO_CACHE_REDIS_URL=redis://cache:6379/0"
+      - "INVENIO_CACHE_TYPE=redis"
+      - "INVENIO_CELERY_BROKER_URL=amqp://guest:guest@mq:5672/"
+      - "INVENIO_CELERY_RESULT_BACKEND=redis://cache:6379/2"
+      - "INVENIO_SEARCH_ELASTIC_HOSTS=['es:9200']"
+      - "INVENIO_SECRET_KEY=CHANGE_ME"
+      {%- if cookiecutter.database == 'postgresql'%}
+      - "INVENIO_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://{{cookiecutter.project_shortname}}:{{cookiecutter.project_shortname}}@db/{{cookiecutter.project_shortname}}"
+      {%- elif cookiecutter.database == 'mysql'%}
+      - "INVENIO_SQLALCHEMY_DATABASE_URI=mysql+pymysql://{{cookiecutter.project_shortname}}:{{cookiecutter.project_shortname}}@db/{{cookiecutter.project_shortname}}"
+      {%- endif %}
+      - "INVENIO_WSGI_PROXIES=2"
+      - "INVENIO_RATELIMIT_STORAGE_URL=redis://cache:6379/3"
```

Comment:
Hmm do you see this more beneficial for users? With this approach it is definitely easier to have/modify config in on place, but I am not sure if we want to give that flexibility.

Reply:
Right, should the invenio.cfg file actually play the role of the .env file? We could maybe merge the 2 together? (rather than have both to limit the number of places where configs can be placed)

Reply:
Oh, this is actually an awesome idea. We limit the number of files too.